test/lib/viewers/ConfigViewer.test.ts

```diff
@@ -28,10 +29,8 @@ describe('ConfigViewer implementations', () => {
 			// ==== ASSERTIONS ====
 			const displayEvents = spyDisplay.getDisplayEvents();
 			expect(displayEvents).toHaveLength(1);
-			expect(displayEvents).toEqual([{
-				type: DisplayEventType.LOG,
-				data: `\nResults formatted as YAML`
-			}]);
+			expect(displayEvents[0].type).toEqual(DisplayEventType.LOG);
+			expect(ansis.strip(displayEvents[0].data)).toEqual(`\n# This is a leading comment\nResults formatted as YAML`);
```

Comment:
For my understanding - I was expecting you would use ansis.dim in the expected text. This test does something different. Can you please please explain? Thanks!

Reply:
`ansis.dim()` is an implementation detail. If we call `ansis.strip()` here, then it doesn't matter how we change the styling in the future, as long as the formatting and content don't change.